MH4U Dex/Images.xcassets/Region Icons/ancestral_steppe-icon.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x",
+      "filename" : "Image.png"
```

Comment:
CC-BY-SA 3.0 includes this (SA) provision:

> ShareAlike — If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original.

which I would take to mean that you'd have to distribute this whole app as CC-BY-SA 3.0.